src/NativeShell/www/apphost.js

```diff
@@ -79,11 +79,11 @@ function getDeviceProfile(profileBuilder, item) {
 };
 
 function getDeviceProfileForVideo(item) {
-    let container = item.Container;
-    let videoTracks = audioTracks = subtitleTracks = [];
+    var container = item.Container;
+    var videoTracks = audioTracks = subtitleTracks = [];
```

Comment:
We should add an eslint file to make sure we aren't using other ES2015+ features.

Reply:
Indeed, but I'd rather not do it right now.